src/amo/components/LandingPage.scss

```diff
@@ -3,3 +3,69 @@
 .LandingPage {
   padding: $padding-page;
 }
+
+.Addon-Header {
+  margin: 10px;
+  max-width: 100%;
+  text-align: center;
+}
+
+.Icon-persona {
+  background: url('~amo/img/icons/themes.svg') 50% 50% no-repeat;
+  background-size: cover;
+  display: inline-block;
+  height: 50px;
+  margin-bottom: 15px;
+  width: 41.5px;
+}
+
+.Icon-extension {
+  background: url('~amo/img/icons/extensions.svg') 50% 50% no-repeat;
+  background-size: cover;
+  display: inline-block;
+  height: 47.8px;
+  margin-bottom: 15px;
+  width: 49px;
+}
+
+.Addon-Header-Text {
+  display: inline-block;
+  margin-left: 20px;
+  text-align: left;
+  width: 250px;
+}
+
+.Addon-Header-Text-Bold {
+  font-size: 20px;
+  font-weight: bold;
+  padding: 0;
+}
+
+.Addon-Header-Text-Content {
+  margin-top: 5px;
+}
+
+.Browse-Button {
+  background-color: white;
+  border-radius: 9px;
+  color: blue;
+  height: 55px;
+  font-size: 12px;
+  margin-left: calc(50% - 150px);
```

Comment:
It would be nice to merge this style with the homepage's buttons. The styles there use `calc` as well but I think they could just be `margin: 0 20px` instead.

You should be able to remove the `border-radius`, `font-size` and `width` styles.